src/components/Accordion/Accordion.js

```diff
@@ -8,11 +8,7 @@ const { prefix } = settings;
 const Accordion = ({ children, className, ...other }) => {
   const classNames = classnames(`${prefix}--accordion`, className);
   return (
-    <ul
-      className={classNames}
-      role="tablist"
-      aria-multiselectable="true"
-      {...other}>
+    <ul className={classNames} {...other}>
```

Comment:
Should we use this PR to swap the order of how the props get spread? e.g.

```suggestion
    <ul {...other} className={classNames}>
```